modular_skyrat/modules/novaya_ert/code/rationpacks.dm

```diff
@@ -1,55 +1,176 @@
-/obj/item/food/nri_course
-	name = "undefined NRI course"
+/obj/item/food/colonial_course
+	name = "undefined colonial course"
 	desc = "Something you shouldn't see. But it's edible."
 	icon = 'modular_skyrat/modules/novaya_ert/icons/rationpack.dmi'
-	icon_state = "main_course"
+	icon_state = "borgir"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 20)
 	tastes = list("crayon powder" = 1)
 	foodtypes = VEGETABLES | GRAIN
 	w_class = WEIGHT_CLASS_SMALL
+	preserved_food = TRUE
 
-/obj/item/food/nri_course/entree
-	name = "combat ration entree"
-	desc = "A vacuum-sealed weather-resistant spaceproof package written in Pan-Slavic, labelled 'завтрак'. It supposedly contains protein-enriched wholewheat biscuits, liver pate and blueberry jam."
-	food_reagents = list(/datum/reagent/consumable/nutriment = 6, /datum/reagent/consumable/nutriment/vitamin = 4)
-	tastes = list("wholewheat biscuits" = 5, "liver pate" = 5, "blueberry jam" = 5)
-	foodtypes = FRUIT | GRAIN | MEAT | BREAKFAST
-
-/obj/item/food/nri_course/main
-	name = "combat ration main course"
-	desc = "A vacuum-sealed weather-resistant spaceproof package written in Pan-Slavic, labelled 'обед'. It supposedly contains snap-to-heat buckwheat porridge and tushonka and snap-to-heat borscht."
-	food_reagents = list(/datum/reagent/consumable/nutriment = 17, /datum/reagent/consumable/nutriment/vitamin = 8)
-	icon_state = "side_dish"
-	tastes = list("hot buckwheat porridge with canned beef" = 5, "hot imperial borscht" = 5)
+/obj/item/food/colonial_course/attack_self(mob/user, modifiers)
+	preserved_food = FALSE
+	icon_state = "[initial(icon_state)]_unwrapped"
```

Comment:
Instead of using this, use `base_icon_state`, that's what we moved into using in the codebase.